src/main/java/org/jenkinsci/plugins/ghprb/GhprbTrigger.java

```diff
@@ -70,7 +72,7 @@
     private static final Logger logger = Logger.getLogger(GhprbTrigger.class.getName());
 
     private static final ExecutorService pool = Executors.newFixedThreadPool(
```

Comment:
I'm not sure how to write tests for this use case.

The thread pool is configurable because I understand one size doesn't fit all.  For my setup, we have GitHub Enterprise with thousands of jobs so I would probably make the count 64 (or just disable it on startup entirely).